src/pages/optimized-for-change/index.zh-hans.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 应变而优
+date: '2018-12-12'
+spoiler: 如何写好一个API？
+---
+
+如何写好一个API？
+
+*优秀* 的 API即好记住又明确，促进代码可读性、准确性和高效性，帮助开发者走向[成功](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
```

Comment:
```suggestion
*优秀*的 API 即好记住又明确，促进代码可读性、准确性和高效性，帮助开发者走向[成功](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
```
Add whitespaces between English and Chinese. See the guideline [here](https://github.com/mzlogin/chinese-copywriting-guidelines#%E4%B8%AD%E8%8B%B1%E6%96%87%E4%B9%8B%E9%97%B4%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC).